File: app/src/main/java/com/example/capstone_donkki/LoginActivity.java
package com.example.capstone_donkki;


import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class LoginActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.act_login);

        EditText et_email = findViewById(R.id.et_email);
        EditText et_pw = findViewById(R.id.et_pw);

        String email = et_email.getText().toString();
        String pw = et_pw.getText().toString();

        //로그인 버튼>>메인 화면으로 연결(DB와 사용자 데이터 비교)
        Button btn_login = findViewById(R.id.btn_login);


        Button btn_missing = findViewById(R.id.btn_missuser);
        Button btn_reg = findViewById(R.id.btn_reg);
        Button btn_google = findViewById(R.id.btn_google);

        btn_login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                //TODO: DB에 있는 이메일/비밀번호 받아오기
                String DB_email = "";
                String DB_pw = "";

                if (email == DB_email && pw == DB_pw) {
                    //사용자 정보 일치>MainActivity 이동
                    Toast.makeText(getApplicationContext(), "어서오세요, 환영합니다!", Toast.LENGTH_SHORT).show();

                    Intent toMain = new Intent(getApplicationContext(), MainActivity.class);
                    startActivity(toMain);
                    finish();

                } else if (email != DB_email || pw != DB_pw) {
                    //사용자 정보 불일치>입력 정보 초기화, 재인증
                    Toast.makeText(getApplicationContext(), "입력된 사용자 정보가 잘못되었습니다.", Toast.LENGTH_SHORT).show();
                    et_email.setText("");
                    et_pw.setText("");
                }

            }
        });

        btn_reg.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent toReg = new Intent(getApplicationContext(), RegisterActivity.class);
                startActivity(toReg);
            }
        });

        btn_google.setOnClickListener(new View.OnClickListener() {
            //TODO: 구글 로그인 기능
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(), "어서오세요, 환영합니다!", Toast.LENGTH_SHORT).show();

                Intent toMain = new Intent(getApplicationContext(), MainActivity.class);
                startActivity(toMain);
                finish();
            }
        });

        btn_missing.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                et_email.setText("");
                et_pw.setText("");

                Intent toMissing = new Intent(getApplicationContext(), MissingUserActivity.class);
                startActivity(toMissing);
            }
        });

    }
}

File: app/src/main/java/com/example/capstone_donkki/LoginActivity.kt
package com.example.capstone_donkki

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.act_login)

        val et_email: EditText = findViewById(R.id.et_email)
        val et_pw: EditText = findViewById(R.id.et_pw)

        val email = et_email.text
        val pw = et_pw.text

        //TODO: DB와 사용자 입력 데이터 비교, 회원 일치 여부 확인
        val btn_login: Button = findViewById(R.id.btn_login)
        btn_login.setOnClickListener {
            val to_main: Intent = Intent(this, MainActivity::class.java)
            //to_main.putExtra()
            startActivity(to_main)
            finish()
        }

        //TODO: 회원정보 분실 화면 연결
        val btn_missing: Button = findViewById(R.id.btn_missing)
        btn_missing.setOnClickListener {
            val to_missing: Intent = Intent(this, MissingActivity::class.java)
            startActivity(to_missing)
            finish()
        }

        //TODO: 회원가입 화면 연결
        val btn_reg: Button = findViewById(R.id.btn_reg)
        btn_reg.setOnClickListener {
            val to_reg: Intent = Intent(this, RegisterActivity::class.java)
            startActivity(to_reg)
            finish()
        }

        //TODO: Google 로그인 API 연결, 구글 로그인 기능
        val btn_google_login: Button = findViewById(R.id.btn_google)
        btn_google_login.setOnClickListener {
            val to_main: Intent = Intent(this, MainActivity::class.java)
            startActivity(to_main)
            finish()
        }
    }
}